lang/ghc-ppc-bootstrap/Portfile

```diff
@@ -17,6 +17,7 @@ description         The Glorious Glasgow Haskell Compilation System
 long_description    This is a package that installs a binary bootstrap GHC compiler.
 
 homepage            https://haskell.org/${canonicalname}
+# This address is unique, hardcoding version here is appropriate:
 master_sites        https://downloads.haskell.org/ghc/7.0.4/krabby/
```

Comment:
Doesn't it seem likely that if the port were ever updated to some other version, the version number in this URL would need to change too?

Reply:
`krabby/` is a unique folder (apparently of someone who made that particular distribution). I do not think we want to create an illusion that this URL is something constant across versions. (It will not work for 7.0.1, for instance.)